examples/js/loaders/3MFLoader.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```

Comment:
It's a javascript thing...

<img width="576" alt="screen shot 2017-08-11 at 4 14 15 pm" src="https://user-images.githubusercontent.com/97088/29230260-2ebb856e-7eb0-11e7-8f04-5cb194dc4bad.png">